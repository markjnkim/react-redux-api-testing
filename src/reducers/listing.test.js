// ./**tests**/reducer_test.js
import reducer from './listing'
// import {initialState} from '../src/redux/reducer'


describe('location reducer', () => {
  it('should return the initial state', () => {
    expect(reducer(undefined, {})).toMatchSnapshot()
  })

  it('should handle DETAIL_SELECTED', () => {
   
    expect(
      reducer({
        listing: {
          selectedListing: null
        },
        location: {
          isLocation: false,
          location: null,
          regions: null,
        },
        dispatch: jest.fn(),
      },
      {
        type: 'DETAIL_SELECTED'
      })
    ).toMatchSnapshot()
  })

  
})