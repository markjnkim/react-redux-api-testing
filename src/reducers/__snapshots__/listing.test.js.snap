// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`location reducer should handle DETAIL_SELECTED 1`] = `
Object {
  "dispatch": [MockFunction],
  "listing": Object {
    "selectedListing": null,
  },
  "location": Object {
    "isLocation": false,
    "location": null,
    "regions": null,
  },
  "selectedListing": undefined,
}
`;

exports[`location reducer should return the initial state 1`] = `
Object {
  "selectedListing": null,
}
`;
